ladybug/src/main/java/nl/nn/ibistesttool/LadybugPipe.java

```diff
@@ -115,15 +117,15 @@ public PipeRunResult doPipe(Object input, IPipeLineSession session) throws PipeR
 
 		Collections.sort(reports, reportNameComparator);
 		long startTime = System.currentTimeMillis();
-		boolean reportGeneratorEnabledOldValue = testTool.getReportGeneratorEnabled();
+		boolean reportGeneratorEnabledOldValue = testTool.isReportGeneratorEnabled();
 		if(enableReportGenerator) {
-			IbisDebuggerAdvice.setEnabled(true);
 			testTool.setReportGeneratorEnabled(true);
```

Comment:
hier nog steeds de oude setReportGeneratorEnabled? wat doet die nu precies?

Reply:
Die werkt gewoon nog als setter. Het enige dat 'testTool.sendReportGeneratorStatusUpdate()' doet is 'testTool.isReportGeneratorEnabled()' aanroepen en het resultaat daarvan doorgeven aan de Debugger ([zie hier](https://github.com/ibissource/ibis-ladybug/pull/20/files#diff-6ff44471a5b5f789a4a459897c11109b)).